pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -447,20 +687,17 @@ func (r *ReconcileWildFlyServer) statefulSetForWildFly(w *wildflyv1alpha1.WildFl
 // loadBalancerForWildFly returns a loadBalancer service
 func (r *ReconcileWildFlyServer) loadBalancerForWildFly(w *wildflyv1alpha1.WildFlyServer) *corev1.Service {
 	labels := labelsForWildFly(w)
-	sessionAffinity := corev1.ServiceAffinityNone
-	if w.Spec.SessionAffinity {
-		sessionAffinity = corev1.ServiceAffinityClientIP
-	}
 	loadBalancer := &corev1.Service{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      loadBalancerServiceName(w),
 			Namespace: w.Namespace,
 			Labels:    labels,
 		},
 		Spec: corev1.ServiceSpec{
-			Type:            corev1.ServiceTypeLoadBalancer,
-			Selector:        labels,
-			SessionAffinity: sessionAffinity,
+			Type:     corev1.ServiceTypeClusterIP,
+			Selector: labels,
+			// SessionAffinity: sessionAffinity,
```

Comment:
Why are you commenting `sessionAffinity`?

Reply:
as it's not relevant for headless service where clustering responses is managed by application itself (in ejb case by the app server)